server/service/user.service.js

```diff
@@ -0,0 +1,13 @@
+import User from './schema/user.schema.js';
+
+export const createUser = async (email, password, firstName) => await new User({ email, password, firstName }).save();
```

Comment:
Mieliśmy wstępnie dodać verificationToken do usera? robimy to od razu tu? Jeżeli tak, to przydałoby się i w tej funkcji go dodawać?

Reply:
Ok